sass/sass_wrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
I think this needs a Google copyright header.

Reply:
Done.

Reply:
I don't see it...

Reply:
Added.